### ISSUE TITLE
Autofocus doesn't work on some devices

### ISSUE THREAD

**Issue by RodrigoPorras** [sawankumarbundelkhandi/edge_detection#15]
First of all, i am very grateful with this package, it was so useful and i saved a lot of time. At the implementation i was getting a crash on some old devices that not support autofocus continuos picture, so i modified the code in kotlin and now i can run the app. here is my modification for someone that maybe need it.

```
if  (pm.hasSystemFeature(PackageManager.FEATURE_CAMERA_AUTOFOCUS) && mCamera!!.parameters.getSupportedFocusModes().contains(Camera.Parameters.FOCUS_MODE_CONTINUOUS_PICTURE)) 

{
            param?.focusMode = Camera.Parameters.FOCUS_MODE_CONTINUOUS_PICTURE
            Log.d(TAG, "enabling autofocus")
        } else {
            Log.d(TAG, "autofocus not available")
        }
```

**Comment by sawankumarbundelkhandi** [comment]
I have added this fix to the plugin code. 
Thanks for the suggestion.

**Comment by sawankumarbundelkhandi** [comment]
I have just published a new version v1.0.5.
Please try it and let me know if you are still facing the same issue.
